sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformationExec.scala

```diff
@@ -412,7 +412,9 @@ case class HiveScriptIOSchema (
     propsMap = propsMap + (serdeConstants.LIST_COLUMN_TYPES -> columnTypesNames)
 
     val properties = new Properties()
-    properties.putAll(propsMap.asJava)
+    // properties.putAll(propsMap.asJava)
+    // see https://github.com/apache/kafka/pull/3647
+    propsMap.foreach{ case (k, v) => properties.put(k, v) }
```

Comment:
Nit: space before brace

Reply:
I think the direct reference should be https://github.com/scala/bug/issues/10418.

Reply:
And we should not leave commented code.